presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/thrift/MockEmptyMetastoreUriFetcher.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore.thrift;
+
+import com.facebook.airlift.http.client.HttpClient;
+import com.facebook.airlift.http.client.Request;
+import org.apache.thrift.TException;
+
+import java.net.URI;
+
+public class MockEmptyMetastoreUriFetcher
+        implements MetastoreUriFetcher
```

Comment:
Move all these testing classes into the unit test itself as private ones. Rename into `TestingXXX` instead of `Mock`